apiv2/src/config/configuration.ts

```diff
@@ -1,18 +1,46 @@
 /*
 Use SENTRY_DSN, SENTRY_ENVIRONMENT, SENTRY_RELEASE
 */
+import { envStr, envInt } from "snu-lib";
+
+function _env<T>(callback: (value: any, fallback?: T) => T, key: string, fallback?: T) {
+    try {
+        return callback(process.env[key], fallback);
+    } catch (error) {
+        console.warn(`Environment ${key}: ${error}`);
+    }
+    return undefined;
+}
+
+// NODE_ENV environment variable is used by :
+// - jest : unit test (NODE_ENV == "test")
+const defaultEnv = process.env.NODE_ENV === "test" ? "test" : "development";
+const environment = _env(envStr, "ENVIRONMENT", defaultEnv);
 
 export default () => ({
+    environment,
+    release: _env(envStr, "RELEASE", "development"),
     httpServer: {
-        port: parseInt(process.env.PORT || "3000"),
+        port: _env(envInt, "PORT", 8086),
     },
     database: {
-        url: process.env.DATABASE_URL || "mongodb://localhost:27017/local_app",
+        url: _env(envStr, "DATABASE_URL", "mongodb://localhost:27017/local_app"), // MONGO_URL in v1
```

Comment:
```suggestion
        url: _env(envStr, "DATABASE_URL", "mongodb://localhost:27017/snu_dev"), // MONGO_URL in v1
```

aussi niveau api v1 pour le defaut